lib/modules/manager/nuget/artifacts.ts

```diff
@@ -55,11 +56,19 @@ async function runDotnetRestore(
     packageFileName,
   );
 
+  // Pass through environment variables that are used by NuGet on Windows.
+  const windowsNuGetEnv = Object.fromEntries(
+    ['PROGRAMFILES', 'PROGRAMFILES(X86)', 'APPDATA', 'LOCALAPPDATA']
+      .map((key) => [key, process.env[key]])
+      .filter(([, value]) => is.nonEmptyString(value)),
+  );
```

Comment:
add them to this list instead
https://github.com/renovatebot/renovate/blob/638deae3d0da4b6a978775c3607f2d06bef3d77e/lib/util/exec/env.ts#L3